package.json

```diff
@@ -22,7 +22,7 @@
     "babel-preset-env": "^1.7.0",
     "babel-preset-es2015": "^6.24.1",
     "babel-register": "^6.26.0",
```

Comment:
For some reason, I still get the following every time I run a task:
```
[11:25:22] Failed to load external module @babel/register
[11:25:22] Requiring external module babel-register
```

It doesn't cause the task to break, but it seems wrong. Would it be possible to fix it so that notice doesn't show up on every task?

Reply:
Based on [gulp issue 1631](https://github.com/gulpjs/gulp/issues/1631) I don't think this is an issue.